compiler/test/data/javascript/class/class.d.kt

```diff
@@ -0,0 +1,3 @@
+
+@JsModule("<RESOLVED_MODULE_NAME>")
```

Comment:
All the generated file actually have to have some standard imports and attributes, please, check addStandardImportsAndAnnotations